src/main/resources/org/opentripplanner/apis/transmodel/schema.graphql

```diff
@@ -1773,6 +1773,8 @@ enum StreetMode {
   car_park
   "Walk to a pickup point along the road, drive to a drop-off point along the road, and walk the rest of the way. This can include various taxi-services or kiss & ride."
   car_pickup
+  "Walk to a car rental point along the road, drive to a drop-off point along the road, and walk the rest of the way. This can include car rental at fixed locations or free-floating services."
```

Comment:
```suggestion
  "Walk to a car rental point along the road, drive to a drop-off point along the road, and walk the rest of the way. This can include car rentals at fixed locations or free-floating services."
```